.classpath

```diff
@@ -46,4 +46,4 @@
 	</classpathentry>
 	<classpathentry kind="con" path="org.eclipse.jdt.junit.JUNIT_CONTAINER/4"/>
 	<classpathentry kind="output" path="target"/>
```

Comment:
What's the reason for this change?

Reply:
If I remove the link to the native windows RXTX library eclipse doesnt find it. I always have to point it to the correct folder and then remove it later again before pushing. Not sure what is going on there

Reply:
So the native library paths are only added when doing an ANT build, so when you start it in eclipse directly its missing I guess?

Reply:
You need to set VM arguments in the Run Configuration for RomRaider.  Without this all native libs will not be found.  Just copy the line from the run.bat file.  Maybe that's why you have odd exceptions at startup.
![image](https://user-images.githubusercontent.com/1688870/79755929-e2105980-82e7-11ea-9d04-e433cb0a5ba5.png)

Reply:
That solved it, thanks!